requirements-dev.txt

```diff
@@ -8,3 +9,5 @@ lxml
 mock
 pytest
 requests-mock
+
+git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.3.0#egg=digitalmarketplace-test-utils==2.3.0
```

Comment:
I think this is a typo - I can't find a version 2.3.0?

Reply:
See https://github.com/alphagov/digitalmarketplace-test-utils/pull/20